eventstore/views.py

```diff
@@ -20,6 +27,89 @@
     PrebirthRegistrationSerializer,
     PublicRegistrationSerializer,
 )
+from ndoh_hub.utils import validate_signature
+
+
+class MessagesViewSet(GenericViewSet):
+    def create(self, request):
+        validate_signature(request)
+        try:
+            webhook_type = request.META.get("HTTP_X_TURN_HOOK_SUBSCRIPTION", "whatsapp")
+            if webhook_type == "whatsapp":
+                for inbound in request.data.get("messages", []):
+                    data = {
+                        "context": inbound.get("context"),
+                        "errors": inbound.get("errors"),
+                        "audio": inbound.get("audio"),
+                        "document": inbound.get("document"),
+                        "image": inbound.get("image"),
+                        "system": inbound.get("system"),
+                        "location": inbound.get("location"),
+                        "text": inbound.get("text"),
+                        "video": inbound.get("video"),
+                        "voice": inbound.get("voice"),
+                    }
+                    Messages.objects.create(
+                        id=inbound["id"],
```

Comment:
This will raise a `KeyError` if the field doesn't exist, which we're not catching, which will mean we'll get an error in Sentry, and response with a 500 status code.

Instead, we should rather catch the key error, and respond with a 400 Bad Request